System/Process/Common.hs

```diff
@@ -0,0 +1,215 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE RecordWildCards #-}
+module System.Process.Common
+    ( CreateProcess (..)
+    , CmdSpec (..)
+    , StdStream (..)
+    , ProcessHandle(..)
+    , ProcessHandle__(..)
+    , withFilePathException
+    , PHANDLE
+    , modifyProcessHandle
+    , withProcessHandle
+    , fd_stdin
+    , fd_stdout
+    , fd_stderr
+    , mbFd
+    , mbPipe
+    , pfdToHandle
+
+-- Avoid a warning on Windows
+#ifdef WINDOWS
+    , CGid (..)
```

Comment:
Ah I see, you've moved the module to hide it from the Users of process but not the developers. Alright then.